includes/class-llms-form-validator.php

```diff
@@ -57,7 +57,15 @@ public function sanitize_field( $posted_value, $field ) {
 
 		$func = isset( $map[ $field['type'] ] ) ? $map[ $field['type'] ] : 'sanitize_text_field';
 
-		return trim( call_user_func( $func, $posted_value ) );
+		// Turn the submitted value into array, so to unify validation of scalar and array posted values.
```

Comment:
beautiful golf, I saw this the other day and thought "WE NEED TO GOLF THIS UPDATE" and you did it!